### PR TITLE
attempt 1 on html proofer, added gem to gemfile and modified script to r...

### PR DESCRIPTION
**Pull request by necampanini**
- added gem to gemfile 
- modified script in .travis.yml to run after a bundle installer
- minor change to the AWS access key and code, not sure why there was a strange symbol there earlier

### CODE REVIEW

**Comment by necampanini** [comment]
So far, html proofer is exiting with 1 because the following errors:

- ./_site/about/index.html
  *  External link https://github.com/poole/issues/new failed: 404 No error
- ./_site/index.html
  *  External link https://codeload.github.com/poole/hyde/zip/v2.1.0 failed: 404 No error

The "#" errors are gone, but I'm not sure what to do here.

**Comment by jmcampanini** [comment]
COOL! it works! replace those links with something like `www.google.com`, the goal is since the content there now is just placeholder crap, we don't care.

it's just really cool to me that it actually works! :-D

File: _posts/2014-01-12-EitS.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+- Simple gem addition in our gemfile
+- and a little modifying our .travis.yml
+- we get htmlproof'ing. 
+
+It broke our travis-ci build because all scripts/things must exit with status 0 to continue. 
+
+The first set of errors mention something about an internal reference to "#" which it didn't like. My solution involves hosing the other posts and just leving this one as a sort of readme. 
+
+  > Hah, an (obvious) Ghost in the Shell reference!
```
Comment (jmcampanini) [inline _posts/2014-01-12-EitS.md]: lol

**Comment by necampanini** [comment]
FIXED. Merging! Looking at MVP email after get some school work done.

**Pull request merged**